src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package Duke.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represent a deadline object with a deadline timing.
+ */
+public class Deadline extends Task {
+    
+    protected LocalDateTime by;
```

Comment:
If the name was clearer, it would be great! Is `dueDate` a possible alternative?